src/did/did_resolver.rs

```diff
@@ -5,6 +5,25 @@ use async_trait::async_trait;
 use crate::{did::did_doc::DIDDoc, error::Result};
 
 /// Represents DID Doc resolver (https://www.w3.org/TR/did-core/#did-resolution).
+#[cfg(feature = "uniffi")]
```

Comment:
Use #[cfg_attr(feature = "uniffi", async_trait))] to avoid struct supplication. https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute

Reply:
We use `#[cfg(feature = "uniffi")]` here as the whole trait is under conditional compilation. 
Not only `async_trait` is added for "uniffi" fearture, but also `Sync` marker needs to be specified.